dione-spark/src/test/scala/com/paypal/dione/spark/index/avro/TestOneLineInBlock.scala

```diff
@@ -0,0 +1,68 @@
+package com.paypal.dione.spark.index.avro
+
+import com.paypal.dione.SparkCleanTestDB
+import com.paypal.dione.hdfs.index.HdfsIndexContants._
+import com.paypal.dione.hdfs.index.HdfsIndexerMetadata
+import com.paypal.dione.hdfs.index.avro.AvroIndexer
+import com.paypal.dione.kvstorage.hadoop.avro.AvroHashBtreeStorageFolderReader
+import com.paypal.dione.spark.avro.btree.SparkAvroBtreeUtils
+import com.paypal.dione.spark.index.{IndexManager, IndexSpec}
+import org.apache.hadoop.fs.Path
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation
+import org.junit.jupiter.api._
+
+object TestOneLineInBlock extends SparkCleanTestDB {
+
+  override val baseTestPath: String = "TestData/TestOneLineInBlock/"
+  override val dbName: String = "TestOneLineInBlock"
+
+  @BeforeAll
+  def initData(): Unit = {
+    import spark.implicits._
+
+    //    val sc = spark.sparkContext
+    //    sc.hadoopConfiguration.setInt("avro.mapred.sync.interval", 100)
```

Comment:
why is that commented out? the test should have interval=1, no?

Reply:
unfortunately, it doesn't work so instead of decreasing the sync.interval i increased the data size to be more than 64k (default value)